Umbraco-StarterKit/Widgets/Grid/Price-table/index.md

```diff
@@ -0,0 +1,35 @@
+---
+versionFrom: 8.0.0
+---
+
+# Price Table
+
+The Price Table, is a great tool for when you want to list multiple products and compare them, for example, different tiers of plans. The Price Table lets you outline the price and what features each tier contains.
+
+Here you can see what it could look like on the front end.
+
+![Price Table](images/Price-Table-frontend1.png) ![Price Table](images/Price-Table-frontend2.png)
```

Comment:
I think only one example is necessary here?

And perhaps, add an example with more features, to give a better effect of what this widget can do?